src/test/java/shop/CartTest.java

```diff
@@ -0,0 +1,41 @@
+package shop;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+public class CartTest {
+    public Cart cart;
+    public RealItem item;
+
+    @BeforeEach
+    public void createCart() {
+        cart = new Cart("test");
+
+        item = new RealItem();
+        item.setName("catfood");
+        item.setPrice(10);
+        item.setWeight(2);
+    }
+
+    @Test
+    void testTotalPrice() {
+        cart.addRealItem(item);
+        cart.addRealItem(item);
+        Assertions.assertEquals((item.getPrice() * 1.2) * 2, cart.getTotalPrice());
```

Comment:
this is magic calculation - can you please move this into class level and give an appropriate name?